### DISCUSSION TITLE
Enchanting for Challenge Modes is Gated by Rods

### DISCUSSION THREAD

**Discussion by NinesCC** [chromiecraft/chromiecraft#5667] [Suggestions]
**This is only regarding Challenge Modes as Hardcore, Ironman, Bloodthirsty or any combination of them:**

Enchanting is the only gated profession since players require different Blacksmith created rods to keep levelling Enchanting to each new tier. It usually is not a problem with a normal character because you are not limited on trading or the AH. On the other hand, Blacksmithing isn't the most popular profession and you have the problem that people with it die or get to 70, reducing the pool of crafters from Challenge Modes.

I don't know how possible it is, but it would be nice if enchanters doing the Challenge Modes could buy these rods from a vendor by exchanging enchanting materials, perhaps? An example of this would be using the disenchanted materials as currency to exchange for the Blacksmithing rod required for the next tier of Enchanting. 

Of course it's just an idea but seems harmless and would allow more people to actually be able to level enchanting.

Edit: Thanks Venom for the help editing this and be more comprehensible :)

**Comment by TylerGoodguy** [comment]
As the original game design suggests, this precedent for Enchanters is somewhat stated by using Tier 1 materials + Blacksmith Rod to build your Enchanted Rod for access to Tier 2 Enchanting. This behavior continues throughout the game. Since your suggestion seems to be a QOL improvement for Challenge Mode players by offering an exchange of materials with NPCs rather than an exchange of gold on the AH with players, I feel like this is satisfactory compromise.

**Comment by Fluctisonant** [comment]
I'm on board with this 100%. As it stands, I'm still the only person in the 3/3 challenge bracket that has bothered to level Blacksmithing enough to be able to provide our 3/3 enchanters with rods and once I finish the challenge a lot of people currently doing the challenge or joining as a new challenger will probably not have the means to obtain the enchanting rods since no one else has bothered to level Blacksmithing (understandably so).

Having an NPC that sells these rods exclusively to challenge players would alleviate this. Implementing it would be a matter of figuring out the logistics regarding how the rods can be redeemed by the challenge players. 

My suggestion would be a gold fee alongside requiring the materials that a Blacksmith would otherwise use to craft the rod.

In the case of the Arcanite Rod, obtaining Arcanite Bars is not easy for challengers, so perhaps the Arcanite Rod could simply cost a bit more gold than the other rods without requiring you to turn in Arcanite Bars. 😊✨

**Reply by TylerGoodguy** [reply]
I think having the rods cost double vendor price would be satisfactory. In the case of the Arcanite Rod, 2.5x vendor cost seems fine to me. Using gold as the exchange and not materials plus gold seems to be much simpler and in favour of the player, so I support this.

**Comment by TheJesterrr** [comment]
Hard agree.  Without having access to the AH or a vast player base, the rod would be very hard to come by as blacksmithing is not an ideal profession for challenge mode, especially for 3/3's. Clearly the profession did not consider the possibility of challenge modes or losing access to these. A player on their own would be unable to go enchanting- as it would require 3 professions to gather the materials, build the rod, then use it. A reality for players doing the rarer combination of challenges like BM/IM
